### PR TITLE
fix: Remove totals table rows from displayed row count

### LINKED ISSUE #1407: Number of rows is counting totals table rows in total
**Description**

image

**Steps to reproduce**

1. create a static table with known number of rows 
    ```
    from deephaven import empty_table
    t = empty_table(1).update("x = i")
    ```
2. note the number of rows you have (1)
3. go to  "Aggregate Columns" from the table sidebar and add any aggregation
4. hover the table or column tooltip

**Expected results**

Shows number of rows as 1.

**Actual results**

Shows number of rows as 2, because it includes the totals table row.

**Versions**

Engine Version: 0.27.0
Web UI Version: 0.44.1
Java Version: 11.0.19
Barrage Version: 0.6.0

### PR DESCRIPTION
**Pull request by GcWan**
Fixes #1407. When aggregations were added, both the table and column tooltips displayed an increased row count to account for the totals table. Rows from the totals table are now subtracted from the row count displayed to users.

### CODE REVIEW

**Comment by GcWan** [comment]
I have read the CLA Document and I hereby sign the CLA

File: packages/dashboard-core-plugins/src/panels/IrisGridPanelTooltip.tsx [hunk 1]
```diff
@@ -15,7 +15,10 @@ interface IrisGridPanelTooltipProps {
 function IrisGridPanelTooltip(props: IrisGridPanelTooltipProps): ReactElement {
   const { model, widgetName, glContainer, description } = props;
 
-  const rowCount = (model?.rowCount ?? 0) - (model?.pendingRowCount ?? 0);
+  const rowCount =
+    (model?.rowCount ?? 0) -
+    (model?.pendingRowCount ?? 0) -
+    (model?.floatingBottomRowCount ?? 0);
```
Comment (mofojed) [inline packages/dashboard-core-plugins/src/panels/IrisGridPanelTooltip.tsx:21]: You'll also need to subtract `floatingTopRowCount`. Right now the total row count is incorrect if you move aggregations to the top:
image

**Review by mofojed — CHANGES_REQUESTED** [review]
Need to handle case where aggregations are on top as well.

**Review by mofojed — APPROVED** [review]

**Comment by mofojed** [comment]
Great work!

**Pull request merged**